src/components/api-reference/example-docs.js

```diff
@@ -56,3 +58,10 @@ export default class ExampleDocs extends React.PureComponent {
     );
   }
 }
+
+ExampleDocs.propTypes = {
+  selectTab: PropTypes.number,
+  tabIndex: PropTypes.number
```

Comment:
Do we even need these two properties for this view...? What function do they perform now? I'm not even sure they're defined, or at least I don't see where else in our codebase we're instantiating `ExampleDocs` with properties...

Reply:
Yeah, pulled these out too - had added these while fixing eslint, then forgot to remove post-refactor